pinot-core/src/main/java/org/apache/pinot/core/io/readerwriter/ForwardIndexReaderWriter.java

```diff
@@ -16,11 +16,29 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pinot.core.io.reader.impl;
+package org.apache.pinot.core.io.readerwriter;
 
+import org.apache.pinot.core.io.reader.ForwardIndexReader;
 import org.apache.pinot.core.io.reader.ReaderContext;
+import org.apache.pinot.core.io.writer.ForwardIndexWriter;
 
 
-public class UnSortedValueReaderContext implements ReaderContext {
+/**
+ * Interface for forward index reader-writer.
+ */
+public interface ForwardIndexReaderWriter extends ForwardIndexReader<ReaderContext>, ForwardIndexWriter {
```

Comment:
I am confused by this change:
* Why is this diff showing as move of an unrelated class?
* The name of the interface is `ForwardIndexReaderWriter`, but it has no read or write api's?

Reply:
1. Somehow git identifies them as a move. It is supposed to replace `BaseSingleColumnSingleValueReaderWriter` and `BaseSingleColumnMultiValueReaderWriter`
2. It extends `ForwardIndexReader` and `ForwardIndexWriter` which contains all the read and write api's